.github/workflows/industrial_ci_action.yml

```
@@ -0,0 +1,27 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on: [push, pull_request]
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
```

`catkin_lint` is [part of the main build job](https://github.com/tylerjw/warehouse_ros_mongo/runs/2150869355?check_suite_focus=true), no need to test this twice. Instead go for Kinetic as before with Travis:
```suggestion
          - {ROS_DISTRO: kinetic, ROS_REPO: main}
```

---

Pushed a rebase of this with that change.

---

Aren't you missing `catkin_lint` now?